Index: application/routes.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from application import app\r\n\r\n@app.route(\"/\")\r\ndef index():\r\n    return 'Hello World'\r\n\r\n
===================================================================
diff --git a/application/routes.py b/application/routes.py
--- a/application/routes.py	
+++ b/application/routes.py	
@@ -1,6 +1,15 @@
-from application import app
+from application import app, db
+from flask import flash, request
 
 @app.route("/")
 def index():
     return 'Hello World'
 
+@app.route("/test", methods=["GET","POST"])
+def testing():
+    # if request.method == "POST" :
+    user = {"user_id":0, 'username':'lex'}
+    return db.users.insert_one(user)
+
+
+
Index: application/__init__.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from flask import Flask\r\n\r\napp = Flask(__name__)\r\napp.config[\"SECRET_KEY\"] = \"cc5349760deee16dab325216f65fc51b0153cf17\"\r\n\r\nfrom application import routes
===================================================================
diff --git a/application/__init__.py b/application/__init__.py
--- a/application/__init__.py	
+++ b/application/__init__.py	
@@ -1,6 +1,13 @@
 from flask import Flask
-
+from flask_pymongo import PyMongo
+from pymongo import MongoClient
 app = Flask(__name__)
-app.config["SECRET_KEY"] = "cc5349760deee16dab325216f65fc51b0153cf17"
+conn = "mongodb+srv://lexdyel-mendez:7taXPnqusWvHb2h5@centro-capital-east.xroqnlt.mongodb.net/?retryWrites=true&w=majority"""
+
+# app.config["MONGO_URI"] = "mongodb+srv://lexdyel-mendez:7taXPnqusWvHb2h5@centro-capital-east.xroqnlt.mongodb.net/?retryWrite3s=true&w=majority"
+
+mongodb_client = MongoClient(conn)
+db_dev = mongodb_client["centro_capital_dev"]
+
 
 from application import routes
\ No newline at end of file
Index: .idea/Centro-Capital.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module type=\"PYTHON_MODULE\" version=\"4\">\r\n  <component name=\"NewModuleRootManager\">\r\n    <content url=\"file://$MODULE_DIR$\" />\r\n    <orderEntry type=\"jdk\" jdkName=\"Python 3.9 (Centro-Capital)\" jdkType=\"Python SDK\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n  </component>\r\n</module>
===================================================================
diff --git a/.idea/Centro-Capital.iml b/.idea/Centro-Capital.iml
--- a/.idea/Centro-Capital.iml	
+++ b/.idea/Centro-Capital.iml	
@@ -2,7 +2,7 @@
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
-    <orderEntry type="jdk" jdkName="Python 3.9 (Centro-Capital)" jdkType="Python SDK" />
+    <orderEntry type="jdk" jdkName="Python 3.9 (CC)" jdkType="Python SDK" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
 </module>
\ No newline at end of file
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectRootManager\" version=\"2\" project-jdk-name=\"Python 3.9 (Centro-Capital)\" project-jdk-type=\"Python SDK\" />\r\n  <component name=\"PyCharmProfessionalAdvertiser\">\r\n    <option name=\"shown\" value=\"true\" />\r\n  </component>\r\n</project>
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	
+++ b/.idea/misc.xml	
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (Centro-Capital)" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (CC)" project-jdk-type="Python SDK" />
   <component name="PyCharmProfessionalAdvertiser">
     <option name="shown" value="true" />
   </component>
